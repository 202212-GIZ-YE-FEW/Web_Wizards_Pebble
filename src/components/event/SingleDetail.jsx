const SingleDetail = ({ text, icon }) => {
    return (
        <div className='lg:w-full md:w-1/2 sm:w-full mb-4 xs:w-full py-2'>
            <h3 className=' font-sans text-black-100 lg:text-lg text-md'>
                <span className='mr-2 inline-block'>{icon}</span>
                <span className='inline-block'>{text}</span>
            </h3>
        </div>
    );
};

export default SingleDetail;
