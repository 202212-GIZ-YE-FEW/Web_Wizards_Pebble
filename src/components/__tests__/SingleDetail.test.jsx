import renderer from "react-test-renderer";

import SingleDetail from "../event/SingleDetail";

it("renders correctly", () => {
    const tree = renderer.create(<SingleDetail />).toJSON();
    expect(tree).toMatchSnapshot();
});
