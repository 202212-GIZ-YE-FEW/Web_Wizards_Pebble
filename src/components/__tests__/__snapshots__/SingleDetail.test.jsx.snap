// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="lg:w-full md:w-1/2 sm:w-full mb-4 xs:w-full"
>
  <h3
    className=" font-sans text-black-100 lg:text-lg text-md"
  >
    <span
      className="mr-2 inline-block"
    />
    <span
      className="inline-block"
    />
  </h3>
</div>
`;
